package.json

```diff
@@ -17,6 +18,7 @@
     "i18next": "^20.1.0",
     "i18next-browser-languagedetector": "^6.1.0",
     "i18next-http-backend": "^1.2.1",
+    "mdtable2json": "^0.1.0",
```

Comment:
Do we still need this package ? Now that we are reading the entire directory instead of table , does it have any utility now ?

Reply:
yes, there is no dependency for the package, will remove it.